### DISCUSSION TITLE
File upload

### DISCUSSION THREAD

**Discussion by amirkhalil22** [pocketbase/pocketbase#2127] [Q&A]
hello, 
I have a pdf file in my C:
so, I want to use my Api to send this file to my collection in pocketbase!
any help!!

**Comment by ganigeorgiev** [comment]
Could you share at least some code how you currently are sending the request?

Did you check the JS example in https://pocketbase.io/docs/files-handling/#uploading-files?

**Comment by amirkhalil22** [comment]
Really I don't do something, this is my Collection  

![image](https://user-images.githubusercontent.com/104743918/226927507-1208cc00-b3a4-4618-b9d7-c24e7d684fd8.png)

So I want to send this file wich is under (C:\Users\Chaker\Desktop\myProjects\CV\CV_KHALIL_Mohamed_Amir.pdf) to pdFile Filed
A+

**Reply by ganigeorgiev** [reply]
Please check the JS example in https://pocketbase.io/docs/files-handling/#uploading-files.
